source/features/make-discussion-sidebar-sticky.tsx

```diff
@@ -5,7 +5,7 @@ import debounce from 'debounce-fn';
 import features from '../libs/features';
 
 function updateStickiness() {
-	const sidebar = select('.discussion-sidebar');
+	const sidebar = select<HTMLElement>('.discussion-sidebar');
```

Comment:
Used the generic `HTMLElement` as I have no idea what kind of element this is supposed to be. I would guess a `<div />`, but not certain.

Reply:
div

Reply:
I'd stick to `HTMLElement`